tests/helpers/module-for-acceptance.js

```diff
@@ -0,0 +1,23 @@
+import { module } from 'qunit'
```

Comment:
I believe we can just remove this file as it depends on qunit which we aren't using.